### PR TITLE
Bound maximum size of random_get buffer.

### PR DESCRIPTION
**Pull request by bryanmcquade**
With this change, random_get will return an error for buffers larger than 64kB.
This is consistent with the JavaScript Crypto.getRandomValues() maximum buffer size.

### CODE REVIEW

File: include/proxy-wasm/limits.h [hunk 1]
```diff
@@ -23,3 +23,9 @@
 #ifndef PROXY_WASM_HOST_MAX_WASM_MEMORY_SIZE_BYTES
 #define PROXY_WASM_HOST_MAX_WASM_MEMORY_SIZE_BYTES (1024 * 1024 * 1024)
 #endif
+
+// Maximum allowed random_get buffer size. This value is consistent with
+// the JavaScript Crypto.getRandomValues() maximum buffer size.
```
Comment (mathetake) [inline include/proxy-wasm/limits.h:28]: could you give us a reference to this JavaScript's behavior?

**Comment by mathetake** [comment]
Want to add tests in https://github.com/proxy-wasm/proxy-wasm-cpp-host/blob/master/test/exports_test.cc ?

File: include/proxy-wasm/limits.h (hunk 1 above)
Comment (bryanmcquade) [reply include/proxy-wasm/limits.h]: Sure, just added a link, thanks!

**Comment by PiotrSikora** [comment]
> Want to add tests in https://github.com/proxy-wasm/proxy-wasm-cpp-host/blob/master/test/exports_test.cc ?

Done, PTAL.

File: include/proxy-wasm/limits.h (hunk 1 above)
Comment (PiotrSikora) [reply include/proxy-wasm/limits.h:28]: FWIW, I'm not sure how relevant this is in this context, since we're not implementing JavaScript API.

**Review by mathetake — APPROVED** [review]

**Pull request merged**